src/Octoshift/Commands/AbortMigration/AbortMigrationCommandArgs.cs

```diff
@@ -0,0 +1,26 @@
+﻿using OctoshiftCLI.Extensions;
+using OctoshiftCLI.Services;
+
+namespace OctoshiftCLI.Commands.AbortMigration;
+
+public class AbortMigrationCommandArgs : CommandArgs
+{
+    public string MigrationId { get; set; }
+    [Secret]
+    public string GithubPat { get; set; }
+
+    public const string REPO_MIGRATION_ID_PREFIX = "RM_";
```

Comment:
We don't need to expose this as a public const, so I would just move this into the `Validate` method. Also since it will be a local `const` it can be a camel cased const variable:
```c#
const string repoMigrationIdPrefix = "RM_";
```

Reply:
True but it was exposed to be used outside of the class but here we don't do that.